curriculum/Chapter-?-Databases.md

```diff
@@ -0,0 +1,30 @@
+# Chapter ?: Databases
+
+## Objectives
+
+After completing this chapter, you will be able to:
+
+- Understand the role and importance of databases in application development.
+- Differentiate between graph, document, and relational-based databases.
+- Implement basic CRUD operations using MongoDB in a NestJS project.
+- Appreciate the nuances of selecting a database type for specific application needs.
+
```

Comment:
Add a section with an example data modeling challenge for each type of DB.

Consider:
- 1 to 2 related entities
- CRUD queries for each